README.md

```diff
@@ -1,21 +1,55 @@
-Chnages:
+# POLYCUB-CONTRACTS
 
-RENAME:
-  - `PENALTY_ADDRESS` to `penaltyAddress`
-  - `LOCKUP_PERIOD_BLOCKS` to `lockupPeriodBlocks`
+#### Before you set up
 
-  - `addPendingClaims()` to `collectPendingRewards()`
+You'll need to install `yarn` and `nodejs` onto your computer, via the command line. 
 
+Yarn Install Guide: https://classic.yarnpkg.com/en/docs/install
 
-Change `claim()` to add second param `limit`, can be set to 0 by default: `claim(bool, uint256)`
+NodeJS install guide: https://nodejs.org/en/download/
 
+#### *Set-up Locally*
+In your terminal, navigate to where you would like this site to live, then run 
+```
+#clone the repo
+git clone https://github.com/CubFinance/polycub-contracts.git
 
----
+#navigate to the directory
+cd polycub-frontends
 
-How contract works?
+# Install the dependencies
+yarn install or npm install
 
-It's standard "vault" masterchef (fork of autofarm), with addition of locked rewards and automatic emission updates.
+```
+# This project demonstrates an advanced Hardhat use case, integrating other tools commonly used alongside Hardhat in the ecosystem.
 
-When user calls `collectPendingRewards()`, `deposit()` or `withdraw()`, pending tokens are stored to array together with unlock block. When `claim()` is called,
-it calculates how many tokens are already past `unlock block` and sends them to the user. if tokens are not unlocked yet (and user specified to claim them too using `true` as first param),
-50% of locked tokens is sent to `penaltyAddress`, 50% to the user.
+Try running some of the following tasks:
+
+```shell
+npx hardhat accounts
```

Comment:
I ran this as an example and gives me some options. What am I supposed to do at this point?